src/api/iiif-metadata/bouwdossier/getBouwdossier.ts

```diff
@@ -0,0 +1,13 @@
+import joinUrl from '../../../app/utils/joinUrl'
+import environment from '../../../environment'
+import { fetchWithToken } from '../../../shared/services/api/api'
+import { Bouwdossier } from './types'
+
+/**
+ * Retrieve iiif-metadata about bouwdossiers
+ *
+ * API documentation: https://api.data.amsterdam.nl/iiif-metadata/bouwdossier
+ */
+// eslint-disable-next-line import/prefer-default-export
+export const getBouwdossier = async (id: string) =>
+  fetchWithToken<Bouwdossier>(joinUrl([environment.API_ROOT, 'iiif-metadata', 'bouwdossier', id]))
```

Comment:
If you want you could already combine some of the parts of the path that are known. For example.
```suggestion
  fetchWithToken<Bouwdossier>(joinUrl([environment.API_ROOT, 'iiif-metadata/bouwdossier', id]))
```

Reply:
What's the difference?

Reply:
Stylistic, but doesn't matter either way.